.github/workflows/deploy.yml

```
@@ -0,0 +1,36 @@
+name: deploy
+
+on:
+  workflow_run:
+    workflows: [documentation.yml]
```

Is this the correct syntax or does it need to be the workflow name?
```suggestion
    workflows: [documentation]
```